src/components/timeseries/ScreenStatusMessage.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { get } from 'lodash';
+
+import blockApi from '../../api/blockApi';
+import dateFormatter from '../../utils/date';
+import ErrorMessageWithIcon from '../error/ErrorMessageWithIcon';
+import Loading from '../Loading';
+
+
+class ScreenStatusMessage extends React.Component {
+
+    constructor() {
+        super();
+
+        this.state = {
+            height: 0,
+            timestamp: '',
+            error: false,
+            loading: true,
+        };
+    }
+
+    componentDidMount = async () => {
+        await this.getBestChainHeight();
+
+        this.setState({
+            loading: false
+        });
+
+        // Constantly execute the method to get the newest block
+        const screenStatusLoopExecution = setInterval(() => {
+            this.getBestChainHeight();
+        }, 60000); // 60 seconds. This is the interval that ElasticSearch takes to ingest data from blocks
+
+        this.setState({ screenStatusLoopExecution });
```

Comment:
Maybe we could store this `screenStatusLoopExecution` is `this.screenStatusLoopExecution`, not in state, given that is not used to render anything in the screen.  What do you think?